std/uni.d

```diff
@@ -622,7 +622,7 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
     // save
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
-
+import std.exception; // enforce
```

Comment:
@wilzbach according to the linked comment even selective imports leak so it wouldn't help to change it.